src/mem/old_api.cc

```diff
@@ -98,7 +99,12 @@ Mem::Stats(StoreEntry * sentry)
         stream << "Suppressed\t" << suppressed << "\n";
     }
 #endif
+    stream << "Current SSL memory usage:\n";
+    Ssl::MallocStats().print(stream);
+    Ssl::ReallocStats().print(stream);
     stream.flush();
+    Ssl::MallocStats().dumpHistogram(sentry);
```

Comment:
I had to split the printing into 2 methods (i.e., print() and dumpHistogram()) because StatHist cannot print directly to stream. We can adjust it, if needed.